cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/KafkaCruiseControlConfig.java

```diff
@@ -444,6 +444,14 @@
   private static final String GOALS_DOC = "A list of case insensitive goals in the order of priority. The high "
       + "priority goals will be executed first.";
 
+  /**
+   * <code>hard.goals</code>
+   */
+  public static final String HARD_GOALS_CONFIG = "hard.goals";
+  private static final String HARD_GOALS_DOC = "A list of case insensitive hard goals in the order of priority. Hard goals"
```

Comment:
`in the order of priority` -> does the order matter here?

Reply:
It does not matter here.